optimum/exporters/onnx/model_configs.py

```diff
@@ -803,10 +803,10 @@ def outputs(self) -> Dict[str, Dict[int, str]]:
             outputs["logits_per_image"] = {0: "image_batch_size", 1: "text_batch_size"}
             outputs["logits_per_text"] = {0: "text_batch_size", 1: "image_batch_size"}
         elif self.task == "zero-shot-object-detection":
-            outputs["logits"] = {0: "batch_size", 1: "num_queries"}
-            outputs["pred_boxes"] = {0: "batch_size", 1: "num_queries"}
+            outputs["logits"] = {0: "image_batch_size", 2: "num_queries"}
```

Comment:
Is the fix changing 1 to 2 here?

Shouldn't it rather be

```
        elif self.task == "zero-shot-object-detection":
            outputs["logits"] = {1: "num_queries"}
            outputs["pred_boxes"] = {1: "num_queries"}
```

if the batch size can't be dynamic?

Reply:
I just kept the batch size for "consistency" with other ONNX configs, but yes it makes sense to remove it. Will do :+1:

Reply:
The change from 1 to 2 is needed because the second dimension of `logits` is not the number of queries but is the number of patches in the images: https://github.com/huggingface/transformers/blob/5bb4430edc7df9f9950d412d98bbe505cc4d328b/src/transformers/models/owlvit/modeling_owlvit.py#L1699

Reply:
Actually ONNX doesn't like when I remove `"image_batch_size"`, it seems to get confused between text and image batch sizes. So let's keep it that way.